Hall_B/AIDAPT/aidapt_toolkit/data_parsers/aidapt_csv_reader_v0.py

```diff
@@ -94,50 +94,49 @@ def save(self, path: str):
         """
         save_dir = Path(path)
         os.makedirs(save_dir)
-        with open(save_dir.joinpath('config.yaml'), 'w') as f:
+        with open(save_dir.joinpath("config.yaml"), "w") as f:
             yaml.safe_dump(self.config, f)
 
     def load_data(self) -> pd.DataFrame:
-        """ Loads all files listed in `config['filepaths']` and concatenates 
+        """Loads all files listed in `config['filepaths']` and concatenates
         them along the `config['axis']` axis
 
         Returns:
             pd.DataFrame: A single DataFrame containing concatenated data
         """
         data_list = []
-        for file in self.config['filepaths']:
-            aidapt_numpy_reader_log.debug(f'Loading {file} ...')
-            data = pd.read_csv(
-                file, 
-                sep=self.config['sep'], 
-                **self.config['read_args'])
+        for file in self.config["filepaths"]:
+            aidapt_numpy_reader_log.debug(f"Loading {file} ...")
+            data = pd.read_csv(file, sep=self.config["sep"], **self.config["read_args"])
             data_list.append(data)
 
         # Check for empty data and return nothing if empty
         if not data_list:
             aidapt_numpy_reader_log.warn(
-                'load_data() returning None. This is probably not what you '
-                'wanted. Ensure that your configuration includes the key '
-                '"filepaths"')
-            return 
-        
-        output = pd.concat(data_list, axis=self.config['axis'], **self.config['concat_args'])
+                "load_data() returning None. This is probably not what you "
+                "wanted. Ensure that your configuration includes the key "
+                '"filepaths"'
+            )
+            return
+
+        output = pd.concat(
+            data_list, axis=self.config["axis"], **self.config["concat_args"]
+        )
         return output
 
     # Unimplemented functions below
     def save_data(self, path: str):
-        aidapt_numpy_reader_log.warning(
-            'save_data() is currently unimplemented.')
+        aidapt_numpy_reader_log.warning("save_data() is currently unimplemented.")
         pass
 
     def load_config(self, path: str):
         aidapt_numpy_reader_log.debug(
-            'load_config() is currently unimplemented. '
-            'Calling load()...')
+            "load_config() is currently unimplemented. " "Calling load()..."
+        )
         return self.load(path)
 
     def save_config(self, path: str):
         aidapt_numpy_reader_log.debug(
-            'save_config() is currently unimplemented.'
-            ' Calling save()...?')
+            "save_config() is currently unimplemented." " Calling save()...?"
```

Comment:
We should remove the internal double quotes here. Functionally it is the same, but it looks odd. I think this happened in multiple files.